server/app.mjs

```diff
@@ -135,8 +135,18 @@ const mount = async (
     process.env.WORKSPACE_DIR ?? process.env.GPTSCRIPT_WORKSPACE_DIR;
   const THREADS_DIR =
     process.env.THREADS_DIR ?? path.join(WORKSPACE_DIR, 'threads');
+
+  // Get the settings file. If it doesn't exist, use default settings instead.
   const settingsLocation = process.env.GPTSCRIPT_SETTINGS_FILE;
-  const settings = JSON.parse(fs.readFileSync(settingsLocation, 'utf8'));
+  let settings = {
+    confirmToolCalls: false,
+    browser: {
+      headless: true,
+    },
+  };
+  if (existsSync(settingsLocation)) {
```

Comment:
Nit: probably more safe to use default setting when file exists and parsing is correct, otherwise fallback to default file. In case the setting file is corrupted...

Reply:
@StrongMonkey Can you take another look? I added some things to defend against malformed files.